app/services/ccms/attribute_value_generator.rb

```diff
@@ -20,6 +20,9 @@ module CCMS
   # method name without the prefix is called on the appropriate object in the options hash, e.g.
   # 'vehicle_registration_number'  will call the registration_number method on options[:vehicle] in order to get the
   # value to insert.
+  #
+  # add a comment to trick circle into running the CCMS tests
```

Comment:
can this comment be removed before merge?

Reply:
Yes, I'll remove the whole commit 👍 
I just wanted to demonstrate the different test-run scenarios for peace of mind 😄